source/connect.txt

```diff
@@ -105,6 +105,46 @@ deployment from which you were disconnected:
 
 .. include:: /includes/steps/activate-connection.rst
 
+Launch MongoDB Shell
+~~~~~~~~~~~~~~~~~~~~
+
+You can connect the
+`MongoDB Shell <https://docs.mongodb.com/mongodb-shell/>`__ or legacy
+:binary:`mongo <mongo>` shell to your current active deployment.
```

Comment:
Is "current active" redundant? I thought you could only have 1 active at a time but am not sure.